cmake/CMakeLists.txt

```diff
@@ -714,31 +714,44 @@ endif()
 
 if(onnxruntime_USE_OPENVINO)
```

Comment:
will we deprecate the option USE_NGRAPH?

Reply:
This is not been considered yet. Let's move this discussion to internal meetings.